miniapps/meshing/mesh-explorer.cpp

```diff
@@ -382,7 +383,8 @@ int main (int argc, char *argv[])
            "f) Find physical point in reference space\n"
            "p) Generate a partitioning\n"
            "o) Reorder elements\n"
-           "S) Save in MFEM format\n"
+           "S) Save in MFEM serial format\n"
+           "D) Save in MFEM parallel format using the current partitioning\n"
```

Comment:
Isn't this option already taken?
`"D) Save as a DataCollection\n"`

Reply:
Maybe it's simpler to just keep `S) Save in MFEM format` and then ask a subsequent question of whether to save in serial or parallel.

Reply:
I just replaced `D` with `T` for this new menu option.